package.json

```diff
@@ -276,58 +277,61 @@
                 "command": "jupyter.replayPylanceLog",
                 "title": "Replay Pylance Log",
                 "category": "Jupyter (Dev)",
-                "enablement": "jupyter.development && notebookType == jupyter-notebook && isWorkspaceTrusted"
+                "enablement": "jupyter.development && notebookType == jupyter-notebook && isWorkspaceTrusted && !jupyter.webExtension"
```

Comment:
This is how I disabled all of the commands in web. By adding a new contextKey : jupyter.webExtension.

Reply:
clearCache is the only command available (which means none in the shipping code)